Foodiea/Profile/VC/ProfileViewController.m

```diff
@@ -30,6 +32,7 @@ - (void)viewDidLoad {
     [super viewDidLoad];
     // Do any additional setup after loading the view.
     _user = [PFUser currentUser];
+    self.manager = [[APIManager alloc] init];
```

Comment:
Let's not alloc the APIManager everywhere we need it to be.

You should implement the manager this as a singleton: https://www.geeksforgeeks.org/singleton-design-pattern/
https://www.galloway.me.uk/tutorials/singleton-classes/